### ISSUE TITLE
Load inspections data from Oracle

### ISSUE THREAD

**Issue by potash** [Chicago/lead-etl#9]
Going forward we should load inspections data from the new Oracle db.

**Comment by geneorama** [comment]
Good idea / point, but it's not in place yet. @tomschenkjr you have a time frame estimate?

**Comment by tomschenkjr** [comment]
I don't. Not sure what that would involve at this point.
